src/strings/csv/en.csv

```diff
@@ -1093,6 +1094,8 @@ SUBSHRUB,Subshrub
 SUBSTRATE,Substrate
 SUBTITLE_GET_STARTED,"To get started, please create your organization now. You'll also be able to set up and manage information on people and species."
 SUBZONE,Subzone
+SUBZONE_PLANTING_COMPLETED_SUCCESS,Subzone marked as planting complete.
+SUBZONE_PLANTING_INCOMPLETED_SUCCESS,Subzone marked as planting incomplete.
```

Comment:
what if user had selected multiple subzones? should this be. `Subzone(s)` instead or ok as-is?

Reply:
Yes, I think suzones(s) would be better